src/pomodoro.js

```diff
@@ -87,7 +88,7 @@ const PomodoroClock = props => {
     }, 200);
 
     return () => clearTimeout(countdown);
-  }, [props, remainingSeconds]);
+  }, [props, remainingSeconds, Sound]);
```

Comment:
not sure but, imho, this `useEffect` is always trigerred cause every render is creating a new instance of `Sound = new Howl`...

So I'm guessing that `config.sound` should be one of the effect's dependencies (cause strings are immutable priimitives not compared by reference) and the `Howl` instance can be created inside the effect scope.